daemon/cmd/daemon.go

```diff
@@ -637,6 +639,8 @@ func newDaemon(ctx context.Context, cleaner *daemonCleanup, params *daemonParams
 
 	d.cgroupManager = manager.NewCgroupManager()
 
+	d.hubbleExporterManager = exporter.NewManager(d.ctx)
```

Comment:
This should probably be a Cell in the Hive. check out `pkg/hive/examples` for the basic structure.